functions/Connect-DbaInstance.ps1

```diff
@@ -462,36 +462,58 @@ function Connect-DbaInstance {
             and within this function leads to the following code path being used.
             All the sub paths inside the following if clause will end with a continue, so the normal code path is not used.
             #>
+
+            # The following line is just added so that we can use AppVayor to run all test with the experimental code path
+            Set-DbatoolsConfig -FullName sql.connection.experimental -Value $true
```

Comment:
FYI, if you want Appveyor to run using this easier to put it in the Test itself.

Reply:
But I want ALL tests, so all commands that run, to use this. I would have to put it in some global Appveyor config?

Reply:
yeah, let me figure this out

Reply:
done! added to \tests\appveyor.pester.ps1